core/src/main/java/com/onelogin/saml2/authn/SubjectConfirmationIssue.java

```diff
@@ -0,0 +1,34 @@
+package com.onelogin.saml2.authn;
+
+import java.util.List;
+
+class SubjectConfirmationIssue {
```

Comment:
Can we move this aux class to /core/src/main/java/com/onelogin/saml2/model/SubjectConfirmationIssue.java ?